near-campaign/src/get_key_balance.rs

```diff
@@ -9,9 +9,9 @@ use crate::*;
 #[near_bindgen]
 impl Campaign {
   pub fn get_key_balance(&self, key: Base58PublicKey) -> U128 {
-    match self.keys.get(&key.into()) {
+    match self.keys.get(&key.0) {
```

Comment:
It will reduce gas usage?

Reply:
not necessarily, the compiler probably will optimize this call away, but safer to do this to be sure